buildSrc/src/main/kotlin/Config.kt

```diff
@@ -1,33 +1,33 @@
 object Config {
```

Comment:
pls remove the whole buildSrc and let's migrate to gradle versions catalog :)

Reply:
done